docs/how-to/HA.md

```diff
@@ -0,0 +1,681 @@
+        -e PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE=2 \
+        -e PERCONA_TEST_POSTGRES_ADDR=${PG_HOST_IP}:5432 \
+        -e PERCONA_TEST_POSTGRES_USERNAME=${PG_USERNAME} \
+        -e PERCONA_TEST_POSTGRES_DBPASSWORD=${PG_PASSWORD} \
+        -e GF_DATABASE_URL=postgres://${GF_USERNAME}:${GF_PASSWORD}@${PG_HOST_IP}:5432/grafana \
+        -e PMM_VM_URL=http://${VM_HOST_IP}:8428 \
+        -e PMM_TEST_HA_ENABLE=1 \
+        -e PMM_TEST_HA_BOOTSTRAP=0 \
+        -e PMM_TEST_HA_NODE_ID=${PMM_PASSIVE2_NODE_ID} \
+        -e PMM_TEST_HA_ADVERTISE_ADDRESS=${PMM_PASSIVE2_IP} \
+        -e PMM_TEST_HA_GOSSIP_PORT=9096 \
+        -e PMM_TEST_HA_RAFT_PORT=9097 \
+        -e PMM_TEST_HA_GRAFANA_GOSSIP_PORT=9094 \
+        -e PMM_TEST_HA_PEERS=${PMM_ACTIVE_IP},${PMM_PASSIVE_IP},${PMM_PASSIVE2_IP} \
+        -v pmm-server-passive-2_data:/srv \
+        ${PMM_DOCKER_IMAGE}
+        ```
+    
+    === "Run services on a seperate instance"
+    
+        ```sh
+        docker run -d \
+        --name ${PMM_PASSIVE2_NODE_ID} \
+        -p 80:80 \
+        -p 443:443 \
+        -p 9094:9094 \
+        -p 9096:9096 \
+        -p 9094:9094/udp \
+        -p 9096:9096/udp \
+        -p 9097:9097 \
+        -e PERCONA_TEST_PMM_DISABLE_BUILTIN_CLICKHOUSE=1 \
+        -e PERCONA_TEST_PMM_DISABLE_BUILTIN_POSTGRES=1 \
+        -e PERCONA_TEST_PMM_CLICKHOUSE_ADDR=${CH_HOST_IP}:9000 \
+        -e PERCONA_TEST_PMM_CLICKHOUSE_DATABASE=pmm \
+        -e PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE=10000 \
+        -e PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE=2 \
+        -e PERCONA_TEST_POSTGRES_ADDR=${PG_HOST_IP}:5432 \
+        -e PERCONA_TEST_POSTGRES_USERNAME=${PG_USERNAME} \
+        -e PERCONA_TEST_POSTGRES_DBPASSWORD=${PG_PASSWORD} \
+        -e GF_DATABASE_URL=postgres://${GF_USERNAME}:${GF_PASSWORD}@${PG_HOST_IP}:5432/grafana \
+        -e PMM_VM_URL=http://${VM_HOST_IP}:8428 \
+        -e PMM_TEST_HA_ENABLE=1 \
+        -e PMM_TEST_HA_BOOTSTRAP=0 \
+        -e PMM_TEST_HA_NODE_ID=${PMM_PASSIVE2_NODE_ID} \
+        -e PMM_TEST_HA_ADVERTISE_ADDRESS=${PMM_PASSIVE2_IP} \
+        -e PMM_TEST_HA_GOSSIP_PORT=9096 \
+        -e PMM_TEST_HA_RAFT_PORT=9097 \
+        -e PMM_TEST_HA_GRAFANA_GOSSIP_PORT=9094 \
+        -e PMM_TEST_HA_PEERS=${PMM_ACTIVE_IP},${PMM_PASSIVE_IP},${PMM_PASSIVE2_IP} \
+        -v /srv/pmm-data:/srv \
+        ${PMM_DOCKER_IMAGE}
+        ```
+        
+
+    !!! note alert alert-primary "Note"
+
+        - Ensure to set the environment variables from Step 1  in each instance where you run these commands.
+        - If you run the service on the same instance, remove the **`-p`** flags.
+        - If you run the service on a separate instance, remove the **`--network`** and **`--ip`** flags.
+
+
+### **Step 7: Running HAProxy**
+
+HAProxy provides high availability for your PMM setup by directing traffic to the current leader server via the `/v1/leaderHealthCheck` endpoint.
+    
+!!! note alert alert-primary "Note"
```

Comment:
please move this to sub-step number 9.  this is where this alert note will be useful, in the command we add in sub-step number 9.  thank you

Reply:
Done.